librocksdb_sys/crocksdb/crocksdb/c.h

```diff
@@ -433,9 +433,27 @@ extern C_ROCKSDB_LIBRARY_API void crocksdb_disable_file_deletions(crocksdb_t* db
 extern C_ROCKSDB_LIBRARY_API void crocksdb_enable_file_deletions(
     crocksdb_t* db, unsigned char force, char** errptr);
 
+extern C_ROCKSDB_LIBRARY_API crocksdb_options_t*
+crocksdb_get_db_options(crocksdb_t* db);
+
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_db_options(crocksdb_t* db,
+                        const char** names,
+                        const char** values,
+                        size_t num_options,
+                        char** errptr);
+
 extern C_ROCKSDB_LIBRARY_API crocksdb_options_t* crocksdb_get_options_cf(
     const crocksdb_t* db, crocksdb_column_family_handle_t* column_family);
 
+extern C_ROCKSDB_LIBRARY_API void
+crocksdb_set_options_cf(crocksdb_t* db,
```

Comment:
crocksdb_set_cf_options?

Reply:
Well, we have a `get_options_cf` above, so I think `set_options_cf` makes it more consistent.